ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageCveSingle.test.js

```diff
@@ -4,8 +4,6 @@ import { verifyColumnManagement } from '../../../helpers/tableHelpers';
 import {
     applyLocalSeverityFilters,
     selectEntityTab,
-    typeAndSelectSearchFilterValue,
```

Comment:
Quick search suggests `typeAndSelectSearchFilterValue` has only the 4 deleted calls in the 2 files.

What are pro and con to delete if from helpers file?

Reply:
Ok good catch, I forgot about checking that. Yes I believe that function (and maybe others) are completely outdated at this point.